packages/@uppy/companion/src/server/provider/zoom/adapter.js

```diff
@@ -65,10 +65,9 @@ exports.getIsFolder = (item) => {
   return !item.file_type
 }
 
-exports.getItemName = (item) => {
-  const start = moment(item.start_time || item.recording_start)
-    .clone()
-    .format('YYYY-MM-DD, kk:mm')
+exports.getItemName = (item, userResponse) => {
+  const start = moment.tz(item.start_time || item.recording_start, userResponse.timezone || 'UTC')
+    .format('YYYY-MM-DD, HH:mm')
```

Comment:
we display datetimes in user's timezone preferably.
Timestamp is also updated to use standard hours, rather than 24 hour time from 1 to 24 (HH instead of kk)